shell/src/main/java/alluxio/shell/command/ChownCommand.java

```diff
@@ -68,13 +75,40 @@ private void chown(AlluxioURI path, String owner, boolean recursive)
     System.out.println("Changed owner of " + path + " to " + owner);
   }
 
+  /**
+   * Changes the owner and group for the directory or file with the path specified in args.
+   * @param path The {@link AlluxioURI} path as the input of the command
+   * @param owner The owner to be updated to the file or directory
+   * @param group The group to be updated to the file or directory
+   * @param recursive Whether change the owner and group recursively
+   */
+  private void chown(AlluxioURI path, String owner, String group, boolean recursive)
+    throws AlluxioException, IOException {
+    SetAttributeOptions options =
+        SetAttributeOptions.defaults().setOwner(owner).setGroup(group).setRecursive(recursive);
+    mFileSystem.setAttribute(path, options);
+    System.out.println("Changed owner:group of " + path + " to " + owner + ":" + group);
```

Comment:
this would be a bit easier to read if you used `System.out.format` and a format string instead